modules/core/04-channel/client/utils/utils.go

```diff
@@ -138,22 +139,22 @@ func QueryLatestConsensusState(
 		return nil, clienttypes.Height{}, clienttypes.Height{}, err
 	}
 
-	clientHeight, ok := clientState.GetLatestHeight().(clienttypes.Height)
```

Comment:
Hmm can we add a query that returns the latest height of a client state?

Then we can use that query to retrieve the latest consensus state correct?

It is worth noting that localhost doesn't have a consensus state so it should be handled correctly here

Reply:
Yeah we can probably add that query which could then be used here. It would add an additional round trip with another query tho. Do you think that's alright? 

The only place this code seems to be used at the moment is in the CLI for transfer in special cases https://github.com/cosmos/ibc-go/blob/main/modules/apps/transfer/client/cli/tx.go#L88